src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -257,17 +257,19 @@
                   {:text (i18n/label :t/open-on-etherscan) :value #(.openURL react/linking url)}])])
 
 (defview transaction-details []
-  (letsubs [{:keys [hash url type] :as transactions} [:wallet.transactions/transaction-details]
+  (letsubs [{:keys [hash url type] :as transaction} [:wallet.transactions/transaction-details]
             confirmations                            [:wallet.transactions.details/confirmations]
             confirmations-progress                   [:wallet.transactions.details/confirmations-progress]]
     [react/view {:style styles/flex}
      [status-bar/status-bar]
      [toolbar/toolbar2 {}
       toolbar/default-nav-back
       [toolbar/content-title (i18n/label :t/transaction-details)]
-      [toolbar/actions (details-action hash url)]]
-     [react/scroll-view {:style transactions.styles/main-section}
-      [details-header transactions]
-      [details-confirmations confirmations confirmations-progress]
-      [react/view {:style transactions.styles/details-separator}]
-      [details-list transactions]]]))
+      (when transaction [toolbar/actions (details-action hash url)])]
```

Comment:
Alignement broken

Reply:
Something weird with those 2 conditionals.

Reply:
no one is part of the toolbar and the other is the main content of the screen.
the first one is to hide the actions menu when the transaction is gone
the second one is to hide the transaction details and replace them with a label when the transaction is gone

Reply:
ok